copilot-chat-frontend.el

```diff
@@ -60,6 +52,7 @@
 Each element must be a `copilot-chat-frontend' struct instance.
 Elements are added in the module that defines each front end.")
 
+(defvar copilot-chat-frontend)
```

Comment:
The `cl-declaim` directly below is a type declaration, so please do not interrupt it with the variable to be declared and do not change the order of the declarations.

Reply:
I've moved it to a more appropriate place.